### ISSUE TITLE
Reduce log spam by discovery.py by lowering logger level from info to debug/warn

### ISSUE THREAD

**Issue by timdonovanuk** [googleapis/google-api-python-client#781]
[This line](https://github.com/googleapis/google-api-python-client/blob/66bb32cc70353296dbf1876ae8c22baf6562cfb9/googleapiclient/discovery.py#L275) in discovery.py:

`logger.info("URL being requested: GET %s", actual_url)`

creates **a lot** of unncessary log spam. I don't think we need to know every time this code makes a request to Google, unless you're debugging. I use this library as part of HomeAssistant and my logs end up looking like (URLs sanitised):

```
2019-11-11 09:44:02 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:02 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/
2019-11-11 09:44:02 INFO (SyncWorker_2) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:02 INFO (SyncWorker_2) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/
2019-11-11 09:44:03 INFO (SyncWorker_9) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:03 INFO (SyncWorker_9) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/
2019-11-11 09:44:03 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:03 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/
2019-11-11 09:44:03 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:04 INFO (SyncWorker_6) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/
2019-11-11 09:44:04 INFO (SyncWorker_3) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
2019-11-11 09:44:04 INFO (SyncWorker_3) [googleapiclient.discovery] URL being requested: GET https://www.googleapis.com/calendar/v3/calendars/

```
 Wouldn't logger.debug be more appropratie? 

Thanks.

**Comment by busunkim96** [comment]
@timdonovanuk Feel free to send a PR if this is of interest!

**Comment by timdonovanuk** [comment]
Thanks for this ..I'm not a developer more just get by hacking a bit of code occasionally so didn't know the correct way to submit PRs to this repo :)

**Comment by parthea** [comment]
Hi @ timdonovanuk, 

I'm going to close this issue as the issue was solved with PR #885.
